dev-libs/glib/glib2-2.54.3.recipe

```diff
@@ -41,11 +40,11 @@ PROVIDES="
 	cmd:glib_compile_schemas$secondaryArchSuffix
 	cmd:gresource$secondaryArchSuffix
 	cmd:gsettings$secondaryArchSuffix
-	lib:libgio_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
-	lib:libglib_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
-	lib:libgmodule_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
-	lib:libgobject_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
-	lib:libgthread_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
+	lib:libgio_2.0$secondaryArchSuffix = 0.5400.3 compat >= 0
```

Comment:
Shall we add a `libVersion` and its `libVersionCompat`?
```diff
 ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="?x86"
+
+libVersion="0.5400.3"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
 	glib2$secondaryArchSuffix = $portVersion compat >= 2
-	lib:libgio_2.0$secondaryArchSuffix = 0.5400.3 compat >= 0
+	lib:libgio_2.0$secondaryArchSuffix = $libVersionCompat
```
(Of course, the purpose of this change is to replace all other occurences of `0.5400.3 compat >= 0` by `$libVersionCompat`.)